generators/app/templates/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "ship-components-",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "grunt server",
+    "lint": "./node_modules/eslint/bin/eslint.js src/",
+    "test": "npm run lint && jest --coverage",
+    "test:update": "npm run test -- -u",
+    "test:watchAll": "npm run test -- --watchAll",
+    "watch:test": "jest --coverage --watch",
+    "coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
+    "build": "npm test && grunt clean && grunt build",
+    "travis": "npm install trevor && trevor"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ship-components/generator-ship-components-boilerplate.git"
+  },
+  "engines": {
+    "node": ">=4.0 <9.0"
+  },
+  "keywords": [
+    "react",
+    "es2015",
+    "es2016",
+    "es2017",
+    "javascript",
+    "javascript-library",
+    "grunt",
+    "webpack",
+    "jest"
+  ],
+  "author": "Sepand Assadi <sepand.assadi@sony.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ship-components/ship-components-tag-input/issues"
+  },
+  "homepage": "https://github.com/ship-components/ship-components-tag-input#readme",
+  "dependencies": {
```

Comment:
Where are these dependencies coming from?

Reply:
The first 5 is by default we probably used them all the time. I removed the rest